notebooks/work-in-progress/fuel-price-imputation/fuel-price-imputation-zane.ipynb

```diff
@@ -0,0 +1,1133 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #31.        "plant_id_eia", 
</pre><p>My first instinct here was "oh no we definitely don't want that", because this picks out individual plants, whereas most of our actual imputations will be of 800 plants the model has never seen before (plus 100 it has seen pieces of). I think the model literally knowing <code>plant_id_eia</code> will only be useful for the second, much smaller group.</p><p>But after more thought, this feature, because it is assigned sequentially over time (right?), probably is a direct proxy for construction date and likely correlates with lots of other temporally dependent things like maybe plant type or location, etc. It might be better to just bring those in directly? I'm undecided here. I guess we can just leave it up to cross validation to decide.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/1696/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>How do you mean <code>plant_id_eia</code> is ordinal in time? You mean as new plants get built the number increments? Is this still an issue if the ID is treated as a categorical feature rather than a numerical one?</p><p> </p><p>I was imagining that this would be helpful in filling in gaps within a plant's fuel price history, but based on what you've seen that seems like a secondary consideration, since almost all the plants either have all their fuel prices, or none of them.</p><p> </p><p>I think a useful geographic feature would be distance to the coalmine associated with the record, which we could estimate as the geographic distance to the centroid of the geometry associated with <code>mine_county_id_fips</code></p><p> </p><p>I'd be surprised if plant age had a lot to do with the fuel price.</p>